test/320_claim_3_test.sh

```diff
@@ -13,12 +14,11 @@ delete_persistence() {
 
 start_suite "Claiming addresses"
 
-weave_on $HOST1 launch-router
-weave_on $HOST2 launch-router $HOST1
+weave_on $HOST1 launch-router --ipalloc-range $UNIVERSE $HOST2
```

Comment:
Without this change, host1 can form a ring without talking to host2, but I want to test what happens when it has to wait for consensus.  I could equally have told it a peer count of 2.